tests/rewrite_test.py

```diff
@@ -30,11 +32,12 @@ def test_change_batch_size(self):
     """Basic test for gde.rewrite.change_batch_size."""
     tf_g = tf.Graph()
     with tf_g.as_default():
-      input_tensor = tf.placeholder(dtype=tf.int32, shape=[32,1],
+      input_tensor = tf.placeholder(dtype=tf.int32, shape=[32, 1],
                                     name="Input")
       result_tensor = input_tensor + 42
     g = gde.Graph(tf_g)
     gde.rewrite.change_batch_size(g, 3, [g[input_tensor.op.name]])
+    print("Graph def is:\n{}".format(g.to_graph_def()))
```

Comment:
did you mean to keep this print statement?

Reply:
Ugh, I thought I got all those. Thanks for catching that one!